src/main/java/org/folio/service/upload/UploadDefinitionService.java

```diff
@@ -47,6 +49,15 @@
    */
   Future<UploadDefinition> updateUploadDefinition(UploadDefinition uploadDefinition);
 
+  /**
+   * Updates {@link UploadDefinition} in database with row blocking
```

Comment:
Please add java doc about exclusive mode, which means the second select cannot be executed until the first one has completed or rolled back.